### PR TITLE
Now supporting namespacing-less wildcards

### PR DESCRIPTION
**Pull request by lindycoder**

### CODE REVIEW

**Comment by godp1301** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ Use it:
 
 - **patterns**
 
-    A list of file paths containing (or not) placeholders (*) to find where are the configuration files.
+    A list of file paths containing (or not) placeholders "(\*)" to find where are the configuration files.
```
Comment (idjaw) [inline README.md]: change to: "to find where the configuration files are located."

File: README.md [hunk 2]
```diff
@@ -30,11 +30,11 @@ Use it:
...
 
     Each placeholder in the path will result in a namespace in the resulting config.  So let's say you have a pattern
 
-        dir1/*/dir2/*.yaml
+        dir1/(*)/dir2/(*).yaml
 
     If this pattern find the file : "dir1/**ns1**/dir2/**file**.yaml" that contains "key: 'value'", the resulting
```
Comment (idjaw) [inline README.md]: nit: s/find/finds/

**Comment by lindycoder** [comment]
@idjaw 's comments addressed

**Comment by idjaw** [comment]
👍

**Pull request merged**